content/projects/data-science-specific/statistical-thinking/_index.md

```diff
@@ -42,38 +42,51 @@ Complete the DataCamp courses [Statistical Thinking in Python Part 1](https://ww
+8. Create a histogram of the CO2 emissions for 1990. Is this the distribution you expected based on your answers from questions 6 and 7? 
+9. Calculate the standard deviation and standard error of the mean for CO2 emissions in 1990. Save each answer in the variables `std_co2_emmisions_1990` and `stderr_co2_emmisions_1990` respectively. How is the standard error different from the standard deviation? 
+10. Create a line graph to show CO2 emissions in Brazil, Russia, China, India, the USA and South Africa over time. What does the graph tell you about the difference and change in C02 emissions in these countries?
+11. What is the mean and standard deviation for land area covered by forest in 1990? Save each answer in the variables `mean_land_area_covered_forest_1990` and `std_land_area_covered_forest_1990` respectively. Why do you think the standard deviation is so large? 
+12. Create histograms for land area covered by forest and percentage of area protected in 1990. Describe the distributions.
+13. Create a scatterplot with a regression line using `seaborn.regplot` to show the relationship between the proportion of land area covered by forest and the percentage of area protected in 2000.
 
   - What is the relationship between these two variables?
   - Describe any patterns in the scatterplot.
   - Do you notice any unusual/extreme values that do not fit the general trend? If you see any unusual values, briefly describe them (Who are they? In what way are they different?).
 
-11. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram.
+14. Since neither forested land area nor protected area is normally distributed, we will need to log transform these variables in order to calculate a correlation coefficient. Log transform the variables and show the transformed distributions in a histogram.
 
-12. Using the `pearsonr` function from the `scipy.stats` module, calculate the Pearson correlation coefficient (and its corresponding p value) to determine the nature of the relationship between proportion of land area covered by forest and the percentage of area protected (as measured in 1990 and log transformed). See `help(pearsonr)` for help on this function.
+15. Using the `pearsonr` function from the `scipy.stats` module, calculate the Pearson correlation coefficient (and its corresponding p value) to determine the nature of the relationship between proportion of land area covered by forest and the percentage of area protected (as measured in 1990 and log transformed). Save this answer in a variable called `pearson_correlation_coefficient_1990`. See `help(pearsonr)` for help on this function.
```

Comment:
There also needs to be a variable name for the p value

Reply:
I'd also like to be more explicit in this question about hypothesis testing.

So perhaps we start the question with:

"We'd like to determine the nature of the relationship between proportion of land area covered by forest and the percentage of area protected (as measured in 1990 and log transformed). 
Start with stating the "null" and "alternative" hypothesis for this inquiry. "

Then continue with the "Using the `pearsonr` function from the..." part